.github/workflows/php-unit-tests.yaml

```
@@ -122,30 +122,30 @@ jobs:
       matrix:
         include:
           - module: mellon
-            php_version: "8.3"      
+            php_version: "8.4"      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)